media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
```

Comment:
Mind the 80 char limit here (see guidelines: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)
```diff
- DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+ DESCRIPTION="qView is an image viewer designed with minimalism and usability \
in mind."
```